qa/tasks/mgr/dashboard/helper.py

```diff
@@ -199,14 +201,18 @@ def tearDownClass(cls):
 
     # pylint: disable=inconsistent-return-statements
     @classmethod
-    def _request(cls, url, method, data=None, params=None):
-        cls.update_base_uri()
+    def _request(cls, url, method, data=None, params=None, version=None):
         url = "{}{}".format(cls._base_uri, url)
         log.info("Request %s to %s", method, url)
         headers = {}
         if cls._token:
             headers['Authorization'] = "Bearer {}".format(cls._token)
 
+        if version:
+            headers['Accept'] = 'application/vnd.ceph.api.v{}+json'.format(version)
+        else:
+            headers['Accept'] = 'application/vnd.ceph.api.v1.0+json'
+
```

Comment:
This looks like done-twice... If the `DEFAULT_VERSION = "1.0"` and `..., version=DEFAULT_VERSION`, then you don't need this?

Additionally, this doesn't leave room for endpoints not supporting the Ceph Versioning... So keeping `version = None` for those would work great for dealing with these 2 cases.